src/rules/order.ts

```diff
@@ -46,6 +46,7 @@ const categories = {
 type Category = keyof typeof categories
 
 const defaultGroups = [
+  'mapped',
```

Comment:
```suggestion
  'internal-imports',
```

How about this?

Reply:
How did you come to the wording `internal-...`?
I came to the name `mapped`, because of what is written on NodeJS docs: https://nodejs.org/api/packages.html#imports

```
Package imports permit mapping to external packages.
                       -------
```

Reply:
Because `mapping` means different in different context, tsconfig `paths` mapping, webpack/rspack resolve `alias` mapping, etc.

Reply:
And is `internal-imports` still a good wording when the NodeJS docs state out that these are `mapping to external packages`?

Reply:
Ah I found more here: https://nodejs.org/api/packages.html#subpath-imports

> In addition to the ["exports"](https://nodejs.org/api/packages.html#exports) field, there is a package "imports" field to create private mappings that only apply to import specifiers from within the package itself.
>
> Entries in the "imports" field must always start with # to ensure they are disambiguated from external package specifiers.

So maybe `private-imports`? 👀

Reply:
> So maybe `private-imports`? 👀

Seems good to me.